task/09-functions-n-closures-tasks.js

```diff
@@ -26,7 +26,10 @@
  *
  */
 function getComposition(f,g) {
-    throw new Error('Not implemented');
+    return function(x) 
+    {
```

Comment:
Curly bracket should be opened on the same line as function or expression declaration

Reply:
Fixed